src/main/java/org/betonquest/betonquest/BetonQuest.java

```diff
@@ -343,6 +336,17 @@ public class BetonQuest extends JavaPlugin {
      */
     private LastExecutionCache lastExecutionCache;
 
+    /**
+     * The BetonQuest plugin constructor.
+     */
+    @SuppressWarnings("PMD.AssignmentToNonFinalStatic")
+    public BetonQuest() {
+        super();
+        instance = this;
+        log = BetonQuestLogger.create(this);
+        pluginTag = ChatColor.GRAY + "[" + ChatColor.DARK_GRAY + getDescription().getName() + ChatColor.GRAY + "]" + ChatColor.RESET + " ";
```

Comment:
```suggestion
        super();
        instance = this;
        log = BetonQuestLogger.create(this);
        pluginTag = ChatColor.GRAY + "[" + ChatColor.DARK_GRAY + getDescription().getName() + ChatColor.GRAY + "]" + ChatColor.RESET + " ";
        eventTypes = new HashMap<>();
        playerDataMap = new ConcurrentHashMap<>();
```

Reply:
Change is explained in the suggestion above

Reply:
reason is explained in the resolved conversion above